wa/workloads/geekbench/__init__.py

```diff
@@ -29,34 +29,25 @@ class Geekbench(ApkUiautoWorkload):
     description = """
     Geekbench provides a comprehensive set of benchmarks engineered to quickly
     and accurately measure processor and memory performance.
-
     http://www.primatelabs.com/geekbench/
-
     From the website:
-
     Designed to make benchmarks easy to run and easy to understand, Geekbench
     takes the guesswork out of producing robust and reliable benchmark results.
-
     Geekbench scores are calibrated against a baseline score of 1,000 (which is
     the score of a single-processor Power Mac G5 @ 1.6GHz). Higher scores are
     better, with double the score indicating double the performance.
-
     The benchmarks fall into one of four categories:
-
         - integer performance.
         - floating point performance.
         - memory performance.
         - stream performance.
-
     Geekbench benchmarks: http://www.primatelabs.com/geekbench/doc/benchmarks.html
-
     Geekbench scoring methedology:
     http://support.primatelabs.com/kb/geekbench/interpreting-geekbench-scores
-
     """
     summary_metrics = ['score', 'multicore_score']
     versions = {
-        '4.0.1': {
+        '4.2.0': {
```

Comment:
Can we not have support for both `4.0.1` and `4.2.0`?
Also this maybe should be in a separate commit?

Reply:
So do you want me to just add another entry for 4.2.0?